src/Models/Scopes/Product/WithProductAttributesScope.php

```diff
@@ -25,7 +25,8 @@ public function apply(Builder $builder, Model $model)
 
         $attributeModel = config('rapidez.models.attribute');
         $attributes = $attributeModel::getCachedWhere(function ($attribute) use ($model) {
-            return in_array($attribute['code'], $model->attributesToSelect);
+            return in_array($attribute['code'], $model->attributesToSelect)
+                && ! in_array($attribute['code'], ['visibility', 'sku', 'type_id', $model->getKeyName()]);
```

Comment:
Why do we need this now and previously not? Can we move this to a separated PR @indykoning? cc @Jade-GG

Reply:
I've cherry-picked the commit and created a PR in #820